src/Resources/assets/js/onboardingWizard/config/wizardTranslations.js

```diff
@@ -35,35 +35,35 @@ export default {
 		text: 'The required action was not performed',
 	},
 	stepPaymentTitle: {
-		text: 'For each method, you can enter a custom title here - it will be displayed on your webshop checkout page.',
+		text: 'For each method, you can enter a custom title here. It will be displayed on your webshop checkout page.',
 	},
 	stepImageUpload: {
-		text: 'Try uploading a custom image for the payment method icon. This will be shown in the wenshop checkout page.',
+		text: 'Try uploading a custom image for the payment method icon. This will be shown in the webshop checkout page.',
 	},
 	stepCountryRestriction: {
 		text: 'Here you can create filters for country specific payment methods - for example, if you want iDEAL only to show for Netherlands customers you choose "Select Countries" and select Netherlands.',
 	},
 	stepPaymentMethod: {
-		text: 'Per method, you can select which Mollie API to use to create payments. Click <a href="https://docs.mollie.com/orders/why-use-orders">here</a> to read about the differences between Orders and Payments API.',
+		text: 'Per method, you can select which Mollie API to use to create payments. Click <a target="_blank" class="mollie-link" href="https://docs.mollie.com/orders/why-use-orders">here</a> to read about the differences between Orders and Payments API.',
 	},
 	stepOrderNumber: {
 		text: 'When using Payments API you may want additional details to help you match payments with customer orders -- you can enter those values here but make sure to use the correct tags provided in the text below',
 	},
 	stepOrdersAPI: {
-		text: 'Orders API <br>This is Mollie suggested API to use for webshops because it allows you to create “orders”. An order contains the personal information of a customer (e.g. address) and products that the customer ordered. When an order is made, a corresponding payment will be created automatically.',
+		text: 'This is Mollie suggested API to use for webshops because it allows you to create “orders”. An order contains the personal information of a customer (e.g. address) and products that the customer ordered. When an order is made, a corresponding payment will be created automatically.',
 	},
 	stepPaymentsAPI: {
 		text: 'Payments API <br>Note: Payments API can not be used for methods such as Klarna',
 	},
 	stepFees: {
-		text: 'In case you have fees that you are passing on to the consumer, you can add them here',
+		text: 'In case you have fees that you are passing on to the consumer, you can add them <a target="_blank" class="mollie-link" href="https://help.mollie.com/hc/en-us/articles/360012564454-Can-I-pass-over-the-costs-for-the-use-of-a-payment-method-to-my-customers-">here</a>',
```

Comment:
do you need class `.mollie-link` to style it? usually it's best to style by class, but those translations may one day be coming from backend (wysiwyg/hardcoded)

Reply:
If i just style <a> tags they will be changed in whole sylius admin panel, so i decided to do class for mollie link + if we use classas in translations, they still can be loaded via |raw ?

Reply:
what about `<a>` tags inside `.mollie__content` or whatever the content container is named?